tools/pd-control.md

```diff
@@ -463,6 +463,76 @@ Protobuf 格式示例：
 }
 ```
 
+### `region store <store_id>`
+
+用于查询某个 store 上面所有的 region。
```

Comment:
region -> **Region** 
Please also update other similar "region"s involved in this PR.